### DISCUSSION TITLE
Voice Assistant for Gamers

### DISCUSSION THREAD

**Discussion by jinsu35** [openai/whisper#1084] [Show and tell]
Hi peeps,

I notice we often need a guide during a game. But you can’t read it and play the game at the same time.

So I built a voice assistant to help you focus on playing.
https://www.gptbro.xyz

Let me know what you think. 

J

**Comment by jck-chan** [comment]
Can't reach it:

ERR_NGROK_3200
Tunnel www.gptbro.xyz not found
